crates/sui-indexer-alt/src/benchmark.rs

```diff
@@ -69,7 +69,7 @@ pub async fn run_benchmark(
         indexer_args,
         client_args,
         indexer_config,
-        false, /* with_genesis */
```

Comment:
This is not consistent -- previously `benchmark` didn't perform bootstrapping, and now you are making it bootstrap. You need to provide a `BootstrapGenesis` here.